realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -142,6 +148,50 @@
     }
 
     /**
+     * Initializes the Realm library. It is required to call this method before interacting with any other of the Realm
```

Comment:
Do we want to go into that much detail? I mean this method is most likely the first method you encounter when using our library so diving into details like logging seems a bit much? Adding a note about the default instance might make sense though